src/fieri/app/models/metrics_runner.rb

```diff
@@ -24,4 +24,8 @@ def cookbook_api_response(params)
   def cookbook_version_api_response(params)
     SupermarketApiRunner.new.cookbook_version_api_response(params['cookbook_name'], params['cookbook_version'])
   end
+
+  def source_repo(cookbook_json)
+    cookbook_json.match(%r{(?<=github.com\/)[\w-]+\/[\w-]+}).to_s
```

Comment:
> I still will need to parse it with a regex because the github url will only accept "some_user/some_repo" when using the contents api.

Ugh. My bad. I missed that nuance to the string needed to Do The Thing™.

![facepalm](https://cloud.githubusercontent.com/assets/517302/23711534/2db644d4-03ee-11e7-8635-076b9b6a2e1e.gif)

So that we don't treat a structured JSON response like a big string, I _think_ I still like ...

```
JSON.parse(cookbook_json)['source_url'].match(%r{(?<=github.com\/)[\w-]+\/[\w-]+}).to_s
```